HW7/hw_7_task_1.py

```diff
@@ -0,0 +1,67 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+# stats: кількість записів
+# add: додати запис
+# delete <name>: видалити запис за іменем (ключем)
+# list: список всіх імен в книзі
+# show <name>: детальна інформація по імені
+#
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+
+phone_book = [{"name": "Stepan Bandera", "phone": "+380501112233"},
+              {"name": "Taras Schevchenko", "phone": "+380672223344"},
+              {"name": "Yevhen Konovalets", "phone": "+380663332255"},
+              {"name": "Lina Kostenko", "phone": "+380634441166"},
+              {"name": "Bohdan Khmelnytskii", "phone": "+380675556677"}]
+
+# function for calculation number of elements
+def com_stats(phone_book):
```

Comment:
nit: better to write info about function inside function
```suggestion
def com_stats(phone_book):
    """
    Return count of phone book items
    """
```